src/screens/SendToken/SendTokenPinConfirmScreen.js

```diff
@@ -93,6 +95,7 @@ class SendTokenPinConfirmScreen extends React.Component<Props, State> {
       isChecking: true,
     }, () => {
       logEvent('transaction_sent', { source: this.source });
+      isLogV2AppEvents() && logEvent('v2_transaction_sent', { source: this.source });
```

Comment:
Source looks like it might not contain any useful data... could we add the from / to symbols here please?